src/pages/resources.mdx

```diff
@@ -0,0 +1,463 @@
+    <ResourceCard
+      subTitle="Brand Center resource page  (w3 ID required)"
+      aspectRatio="2:1"
+      href="https://www.ibm.com/brand/resources"
+      >
+
+![](/images/bee.png)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## External resources
+
+<Row>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Thinking with type: baseline grid"
+      aspectRatio="2:1"
+      href="http://thinkingwithtype.com/grid/#baseline-grid"
+      >
+
+![]()
+
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Grid systems in graphic design"
+      aspectRatio="2:1"
+      href="https://www.amazon.com/Grid-Systems-Graphic-Design-Communication/dp/3721201450"
+      >
+
+![]()
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Logo Requests
+
+### IBM Logo use by IBMers
+
+- The IBM logo is the intellectual property of IBM and it is one of our most valuable corporate assets.  As IBMers, it is our responsibility to protect it. By requesting and downloading the IBM logo image files, you are acknowledging that you understand and will adhere to the IBM Logo guidelines, including:
+
+- You are using the IBM logo for IBM sanctioned business purposes. Any other use of this logo image is strictly prohibited.
+
+- You will not distribute the IBM logo file to third parties for their use. All third-party logo requests must be approved. Submit third-party requests through the [Logo Request Tool](https://prdpcrhibmbl01.w3-969.ibm.com/marketing/logotool/logotool.nsf/BrandingLogoHome?Openform) (w3 access is required).
+
+- The IBM logo may not be used for any personal or commercial purposes. Email the following information to [branding@us.ibm.com]()
+
+  * Your full name
+
+  * Your IBM email address
+
+  * A description of how you intend to use the IBM logo files
+
+
+### Agency and third-party requests for the IBM logo use by outside agency or third-party
+
+We understand that our agency partners may require the IBM logo files in order to create materials on behalf of IBM. Please ask your IBM sponsor or client for the image files. They may also submit a request on your behalf using the instructions above.
+
+<Row className="resource-card-group">
+
+<Column colMd={4} colLg={4} noGutterSm>
+<ResourceCard
+subTitle="IBM 8-bar logo request tool in Brand Center"
+href="https://prdpcrhibmbl01.w3-969.ibm.com/marketing/logotool/logotool.nsf/BrandingLogoHome?Openform"
+>
+
+![](/images/ibm.svg)
+
+</ResourceCard>
+</Column>
+
+<Column colMd={4} colLg={4} noGutterSm>
+<ResourceCard
+subTitle="IBM Business Partner marks in Brand Center"
+href="https://www.ibm.com/brand/ibm-logos/business-partners#resources"
+>
+
+![](/images/bee.png)
+
+</ResourceCard>
+</Column>
+
+<Column colMd={4} colLg={4} noGutterSm>
+<ResourceCard
+subTitle="IBM rebus request tool in Brand Center"
+href="https://www.ibm.com/brand/ibm-logos/business-partners#resources"
+disabled
+>
+
+![](/images/_rebus.png)
```

Comment:
this image is showing up the wrong size, take a look at this page `/design/language/elements/ibm-logos/rebus/` to see how we made it the corrrect size

Reply:
fixed it.